schema.sql

```diff
@@ -22,3 +22,8 @@ ALTER TABLE animals ALTER COLUMN id SET DATA TYPE INTEGER;
 ALTER TABLE animals ALTER COLUMN id SET NOT NULL;
 ALTER TABLE animals ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1);
 ALTER TABLE animals ADD PRIMARY KEY (id);
+
+CREATE TABLE vets (id SERIAL PRIMARY KEY, name VARCHAR(50), age INTEGER, date_of_graduation DATE);
+
+CREATE TABLE specializations (vet_id INTEGER REFERENCES vets(id), species_id INTEGER REFERENCES species(id), PRIMARY KEY (vet_id, species_id));
+CREATE TABLE visits (id SERIAL PRIMARY KEY, animal_id INTEGER, vet_id INTEGER, visit_date DATE, FOREIGN KEY (animal_id) REFERENCES animals(id), FOREIGN KEY (vet_id) REFERENCES vets(id));
```

Comment:
![image](https://github.com/edgardsoza/vet-clinic-database/assets/93039359/57f32bf2-eb4a-434c-ab15-791f7a0df9ca)

- [ ] It would be nice if you address this issue shown in the image above because this is stopping me from creating the visits table 😨 Kindly fix it to make your project follow the requirements 👍